### DISCUSSION TITLE
umi 3.5 热更新速度很慢 求个解决方案

### DISCUSSION THREAD

**Discussion by LifeIsLikeABook** [umijs/umi#12497] [Q&A]
umi3.5的项目 项目体积中上 热更新速度很慢 打印个log要等待二分钟以上，开启了MFSU方案没有任何作用，查询了GPT-4o给的解决方案也是不管用解决不了，特此来求助大佬给个解决方案

**Comment by fz6m** [comment]
umi 3 不维护了，请升级 umi 4 。

如果无法升级，并且热更新速度很慢，单独创建一个仅供开发时使用的配置，将 `routes` 配置为仅限当前开发的 layout 和 route ，从而最大化减少启动项目需要加载的 JS ，即可缓解热更新速度很慢的问题。长期此项目必须拆分、重构、升级，否则永远将保持很慢的速度。
